### DISCUSSION TITLE
webGIS-OptionBox ImageServer RasterFunctions

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#171] [General]
Hallo Jürgen,

wir verwenden ein (1) ArcGIS-Server ImageService [Hoehen] mit einigen RasterFunctions, mit denen man das Höhenmodell unterschiedlich darstellen kann: Exposition, Hangneigung ...
Zu sehen ist das im Digitalen Atlas: "[Höhen- und Geländedarstellung](https://gis.stmk.gv.at/wgportal/atlasmobile/map/H%C3%B6hendarstellung%20-%20Gel%C3%A4ndeinformation/H%C3%B6hen-%20Gel%C3%A4ndedarstellung)" 

im DV-Container Höhen- und Geländedarstellung in der dv-Gruppe Geländeinformation sind diese unterschiedlichen Darstellungen drin und mit Optionboxes auswählbar ... leider funktioniert das nicht wie gewohnt - eher wie Checkboxen.

Wie gesagt, dahinter ist ein AGS-IS-Service, das im webGIS öfter eingebunden ist, unterschiedlich parametrisiert über die Eingabe der RasterFunctions.

CMS-kommt per Email.

**Comment by gis-eni** [comment]
Bug behoben
